profiling/src/php_ffi.c

```diff
@@ -75,6 +75,12 @@ void datadog_php_profiling_copy_string_view_into_zval(zval *dest, zai_string_vie
     }
 }
 
+void datadog_php_profiling_copy_long_into_zval(zval *dest, long num) {
```

Comment:
Can we use the newer `ddog_php_prof_` prefix instead of `datadog_php_profiling_`?